catroid/src/main/res/values/strings.xml

```diff
@@ -1078,12 +1076,12 @@
     <!-- Physics -->
     <string name="default_project_name_physics">My first physics program</string>
```

Comment:
I think `default_project_name_physics` is actually not used anymore.

Reply:
It actually is.